crates/ruff_linter/src/rules/pyupgrade/rules/unnecessary_default_type_args.rs

```diff
@@ -45,11 +45,10 @@ use crate::checkers::ast::Checker;
 /// ```
 ///
 /// ## References
-///
 /// - [PEP 696 – Type Defaults for Type Parameters](https://peps.python.org/pep-0696/)
 /// - [Annotating generators and coroutines](https://docs.python.org/3.13/library/typing.html#annotating-generators-and-coroutines)
-/// - [typing.Generator](https://docs.python.org/3.13/library/typing.html#typing.Generator)
-/// - [typing.AsyncGenerator](https://docs.python.org/3.13/library/typing.html#typing.AsyncGenerator)
+/// - [Python documentation: `typing.Generator`](https://docs.python.org/3.13/library/typing.html#typing.Generator)
```

Comment:
I think we should just link to the latest version of these docs rather than specifically the docs for Python 3.13. Currently the 3.13 docs are the same as the latest version but that won't always be the case

```suggestion
/// - [Python documentation: `typing.Generator`](https://docs.python.org/3/library/typing.html#typing.Generator)
```

Reply:
Absolutely, overlooked this.